_includes/google-search.html

```diff
@@ -0,0 +1,12 @@
+<script>
+    (function() {
+      var cx = 'e20a3e7ce37c2454c';
+      var gcse = document.createElement('script');
+      gcse.type = 'text/javascript';
+      gcse.async = true;
+      gcse.src = 'https://cse.google.com/cse.js?cx=' + cx;
```

Comment:
I haven't yet looked at how feasible this is with other search engines, but I'm curious if we'd prefer to use alternatives like DDG or similar?

Reply:
I have half a mind to spend a few hours doing a custom very basic search with no external dependencies (after this is merged) by reusing the .json index generating trick in this PR. We may lose some search functionality like synonyms though.

Reply:
I tried two versions of it, and it was hard (you can look at previous commits on the branch) -- the tutorials online didn't work out of the box, and I find that the quality of google search is quite good actually! If you want to try it, you definitely can :)

Reply:
But we can leave that to a different commit.